### ISSUE TITLE
Insufficient Color Contrast: "Learn More" button

### ISSUE THREAD

**Issue by MelSumner** [emberjs/website#3164]
**Context:** 
The EmberConf CTA "learn more" button does not have sufficient color contrast to pass WCAG 2.0 AA standards. Users who are color-blind would find it difficult, if not impossible, to distinguish between the button background and the text. 

**To Fix:** 
Ensure that the solution has a color contrast ratio of 4.5:1. Color contrast tools (like https://webaim.org/resources/contrastchecker/) can help with this. 

**Screenshot:** 
![image](https://user-images.githubusercontent.com/4587451/35194534-c6e0e542-fe7a-11e7-83e9-9d49bb36e6da.png)
